data_pipeline/surveymonkey/surveymonkey_etl.py

```diff
@@ -30,13 +30,71 @@ def get_survey_list(access_token: str) -> list:
 def get_survey_question_details(access_token: str, survey_id: str) -> list:
     headers = get_surveymonkey_api_headers(access_token)
     response = requests.get(
-        f'https://api.surveymonkey.com/v3/surveys/{survey_id}/details',
+        f"https://api.surveymonkey.com/v3/surveys/{survey_id}/details",
         headers=headers
     )
     reponse_json = response.json()
     return reponse_json
 
 
+def get_no_answer_options_for_question_json() -> dict:
+    return {
+        "id": None,
+        "text": None,
+        "type": None,
+        "weight": None
+    }
+
+
+def parse_answer_options_in_question_answer_json(
+    question_answer_response_json: dict,
+    option_key: str
```

Comment:
This function doesn't seem to be using `option_key` apart from using it as a key for `question_answer_response_json`. Maybe in that case it would make sense to just pass `question_answer_response_json[option_key]` to the function?